src/jdk.jlink/share/classes/jdk/tools/jmod/JmodTask.java

```diff
@@ -767,6 +767,10 @@ void processSection(JmodOutputStream out, Section section, List<Path> paths)
         void processSection(JmodOutputStream out, Section section, Path path)
             throws IOException
         {
+            // Keep a sorted set of files to be processed, so that the jmod
+            // content is reproducible as Files.walkFileTree order is not defined
+            SortedMap<String, Path> filesToProcess  = new TreeMap<String, Path>();
```

Comment:
Nit: there seems to be two whitespaces (instead of one) before the `=` sign on this line.

Reply:
thanks, fixed